docs/user-guide/project.md

```diff
@@ -28,120 +28,218 @@ A user menu with different actions is accessible for each member of the project
 
-In both cases it will open the property form where you have to define a label and optional a comment. If you have selected an existing property, you can also change the label and comment. Be careful in this case, because it can have an effect on other resource classes that use the same property! 
+If you want to define a property which can have more than one value, you should tick `Multiple values?`. For the number of pages of a book this does not make sense, but in the case you want to define a property describing which people are mentioned in your *Book*, the option multiple values is likely to be needed.
 
-The property type can't be changed here. Depending on the property type you have to set additional attributes, e.g. in a resource link property you have to select a resource class to which the property will point to. Similar in case of a list property where you have to select the corresponding list. In both cases a the linked resource class or the list should already exist.
+Now you should see the new property in the box as seen below:
+![Layout after the creation of the property.](../assets/images/Fig17.png)
 
-![Data model editor | Step 4: Define property's label and comment and set the cardinality corresponding to the resource class.](../assets/images/data-model-property-create.png)*Data model editor | Step 4: Define property's label and comment and set the cardinality corresponding to the resource class.*
+### Define Lists
+One of the possible property types to choose from is *List*. Lists are very useful if you want to use controlled vocabulary to describe something. Typical examples are keywords. In your book example it may be useful to define a property which describes to which category of literature your pdf of a book belongs. **Before** you can add a property of type *List* to your data model, you have to define this list. For the definiton of a list you have to change to the `Lists` tab:
+![List tab.](../assets/images/Fig34.png)
 
-For each property in a resource class you can define some rules. For example you can define if one of the property can have `multiple` values or if a property is a mandatory (`requiered`) field.
+Click `Create your first list`. If there is already a list defined, click `New list`. A new window pops up where you have to enter a name for your list (`List label`) and a short description, then click `Create`.
+![Create new list.](../assets/images/Fig35.png)
 
-After adding more than one property to a resource class you can change the displayed order by drag'n'drop the properties in the list.
+As soon as the list is created you can continue with the definition of your data model. The single list items you can define later. How to do this will be explained below. We first focus on the definition of a list property in the data model. Currently there is only one option for the display of a property of type *List*, namely *Dropdown*. It is capable to display flat as well as hierarchical lists.
```

Comment:
I corrected multiple things, you can just copy and paste:
As soon as the list is created you can continue with the definition of your data model. You can define the individual list items later. How to do this will be explained below. We first focus on the definition of a list property in the data model. Currently there is only one option for displaying a property of type *List*, namely *Dropdown*. It is capable of displaying flat as well as hierarchical lists.